Makefile

```diff
@@ -1,18 +1,36 @@
-BUILD_NUMBER := 1
-CP_VERSION := 3.3.0-SNAPSHOT
+# You can override vars like REPOSITORY in a local.make file
+-include local.make
+
+# Bump this on subsequent build, reset on new version or public release. Inherit from env for CI builds.
+BUILD_NUMBER ?= 1
+
+CONFLUENT_MAJOR_VERSION ?= 3
+CONFLUENT_MINOR_VERSION ?= 3
+CONFLUENT_PATCH_VERSION ?= 0
+
+CONFLUENT_VERSION ?= ${CONFLUENT_MAJOR_VERSION}.${CONFLUENT_MINOR_VERSION}.${CONFLUENT_PATCH_VERSION}
```

Comment:
Where does `SNAPSHOT` get added? This seems relevant since below here you define `VERSION` which is going to be used as the docker image tag. Or is the expectation that you override all of `CONFLUENT_MAJOR_VERSION`, `CONFLUENT_MINOR_VERSION`, `CONFLUENT_PATCH_VERSION`, `CONFLUENT_VERSION`, and `KAFKA_VERSION` if you override anything? Would an empty-by-default but optional `SNAPSHOT` or `SUFFIX` or something like that to cover `SNAPSHOT`, `alpha`, `beta`, etc make sense here?